### PR TITLE
routing admin/customer basic

### PR DESCRIPTION
**Pull request by phillipet**
Add more  util.getUserType function use for routing admin/customer
Adjust routing customer correctly with new scenario 
Prevent customer use auth-token route to /admin path
Route admin basic ( only handle routing with path = /admin, others admin paths will be implement later )

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -22,23 +22,39 @@ export default function({store}) {
...
-        to.path === '/customer/login' ||
-        (to.path !== '/customer/login' && to.path !== '/customer') // prevent route to customer page
-      ) {
-        next()
-      } else next('/customer/login')
+
+      // user is ADMIN
+    } else if (userType === 'a') {
+      // route to admin home page
+      if (to.path === '/admin') next()
```
Comment (hemvanh) [inline src/router/index.js]: if (to.path === '/admin') next()
---> same with:  else next('/admin')
??

File: src/router/index.js [hunk 2]
```diff
@@ -22,23 +22,39 @@ export default function({store}) {
...
+      else next('/admin')
+
+      // user is ANONYMOUS
+    } else if (userType === false) {
+      // recognize page of user by path to route
+      if (to.path.indexOf('customer') > -1) {
```
Comment (hemvanh) [inline src/router/index.js]: what if the url is "/admin/**customers**" ?

File: src/router/index.js [hunk 3]
```diff
@@ -22,23 +22,39 @@ export default function({store}) {
...
+        if (
+          to.path === '/customer/register' ||
+          to.path === '/customer/login' ||
+          (to.path !== '/customer/login' && to.path !== '/customer') // prevent route to customer page
+        ) {
+          next()
+        } else next('/customer/login')
+      } else {
+        // routing to ADMIN login page
+        if (to.path === '/admin/login') next()
```
Comment (hemvanh) [inline src/router/index.js]: if (to.path === '/admin/login') next()
--> same with : **else next('/admin/login')**
??

File: src/util/common.js [hunk 4]
```diff
@@ -63,3 +63,17 @@ export const _procError = err => {
 export function isAuth() {
   return !!localStorage.getItem('auth-token')
 }
+
+export function getUserType() {
+  let token = localStorage.getItem('auth-token')
+  let userType = token.substr(token.length - 1)
+  switch (userType) {
```
Comment (hemvanh) [inline src/util/common.js]: how about just: return userType  ?

**Review by hemvanh — CHANGES_REQUESTED** [review]
tune the code abit more

**Review by hemvanh — CHANGES_REQUESTED** [review]
forgot to update ChangeLogs ?

File: src/util/common.js (hunk 4 above)
Comment (phillipet) [reply src/util/common.js]: ok, I will only return last char, checking char will be handled by router

File: src/router/index.js (hunk 2 above)
Comment (phillipet) [reply src/router/index.js]: it go to 404 page http://prntscr.com/l5572p

File: src/util/common.js (hunk 4 above)
Comment (hemvanh) [reply src/util/common.js]: yea coz userType is null , then !null also = true

Comment (hemvanh) [reply src/util/common.js]: so the outside checking we still can keep as the same, no need to change anything

File: src/router/index.js (hunk 2 above)
Comment (hemvanh) [reply src/router/index.js]: /admin/customers will be there in the future

**Comment by phillipet** [comment]
I done manual testing for new routing algorithm
Test cases at : https://trello.com/c/UdDS06iH/102-routing-admin-check-for-localstorage-auth-token5#comment-5bc07b6a6f3ee22f5d2115e1

**Pull request closed without merging**